### DISCUSSION TITLE
"WebSocket /cqhttp/ws" 403

### DISCUSSION THREAD

**Discussion by liuzimo** [nonebot/discussions#79] [问答]
![image](https://user-images.githubusercontent.com/25811500/149260014-b495bb1e-24da-4f73-bd07-664f73e48970.png)

![image](https://user-images.githubusercontent.com/25811500/149260083-23bf940a-370a-4967-b304-0e177dc6f732.png)

**Comment by synodriver** [comment]
设置了token吗？

**Comment by liuzimo** [comment]
> 

仅仅是nb create  后修改了6666的端口，因为我的go-cqhttp是6666

**Comment by liuzimo** [comment]
全部是今天最新的版本

**Comment by yanyongyu** [comment]
上报地址错误，请阅读新版文档 https://adapter-onebot.netlify.app/

**Comment by synodriver** [comment]
记错了 你的是onebotadapter啊

**Comment by liuzimo** [comment]
好繁琐啊

**Comment by liuzimo** [comment]
一个版本一个方式 都不统一下

**Comment by yanyongyu** [comment]
cqhttp已经改名onebot v11，这不是nb统一不统一的事 https://11.onebot.dev/

**Reply by gwozai** [reply]
已解决，谢谢
